_episodes/02-starting-with-data.md

```diff
@@ -272,6 +272,12 @@ surveys_df.head() # The head() method displays the first several lines of a file
 ~~~
 {: .output}
 
+We can also load the csv file in to the Pandas dataframe directly from the URL without downloading using the below syntax. 
+~~~
+surveys_df=pd.read_csv('https://ndownloader.figshare.com/files/2292172')
+~~~
+{: .language-python}
+
```

Comment:
I like this addition, because learners may indeed benefit greatly from knowing that dataframes can be created directly from an online data set. However, in my opinion this is not essential to the flow of the lesson. That makes this information a perfect fit for a callout box.

Callouts can be added to a lesson as "special blockquotes" ([see the full documentation]()), applying the `.callout` class as follows:

```
> ## Callout Title
> 
> Callout box contents.
>
{: .callout}
```

If you can adjust this contribution to format the additional information as a callout, I would be happy to merge.